CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## Upcoming
+
+### Design changes
```

Comment:
If all points in section is regarding a single component, let the component name be the heading.

Reply:
Fixed!